src/IonParserBinaryRaw.ts

```diff
@@ -464,7 +464,7 @@ export class ParserBinaryRaw {
                 } else if (this._len <= 4) {//32 bits of representation after shifts
                     this._curr = this.readUnsignedInt();
                 } else {
-                    this._curr = this.readUnsignedLongInt();
+                    throw new Error("Cant handle over 32 bits of int precision.");
```

Comment:
This error message is a bit terse--it would require the (perhaps non-technical) user to read through the associated stack trace for more context. 

There's already a code path for detecting oversized ints in [`ParserBinaryRaw#readUnsignedInt()`](https://github.com/amzn/ion-js/blob/development/src/IonParserBinaryRaw.ts#L254-L259) -- should we just update the error message there and remove the `if` check here on `this._len`?

Reply:
On this branch specifically were returning longint to apis expecting int so there might be other paths that reach this point. Supporting more bytes here will really require a number tower.